app/components/gist/files_component.html.erb

```diff
@@ -1,5 +1,5 @@
 <% gist.gist_files.each do |file| %>
   <div class="highlight">
-   <%= helpers.rouge(file.raw_code, "python").html_safe %>
+   <%= rouge(file.raw_code, file.language.rouge_name).html_safe %>
```

Comment:
this fail on my side, I get undefined method rouge. that's why I was fetching it from helpers 😢

Reply:
view components are encapsulated so dont have access to helpers. i included the application helper in the viewcomponent.rb file so all good!